docs/semgrep-ci.md

```diff
@@ -1,180 +1,283 @@
 ---
 append_help_link: true
+meta_description: |-
+  Semgrep CI is a specialized Docker image for running Semgrep in CI environments. It can also be used stand-alone or connected with Semgrep App for centralized rule and findings management. Rapidly scan every commit and block new bugs with Semgrep CI.
 ---
 
 # Semgrep CI
 
-[Semgrep CI](https://github.com/returntocorp/semgrep-action)
-is a wrapper around
-[Semgrep CLI](https://github.com/returntocorp/semgrep)
-that adds convenient features for use in CI environments,
-such as in GitHub Actions or GitLab CI.
+[Semgrep CI](https://github.com/returntocorp/semgrep-action) (aka Semgrep Action or `semgrep-agent`) is a specialized Docker image for running Semgrep in CI environments. It can also be used stand-alone or connected with [Semgrep App](https://semgrep.dev/login) for centralized rule and findings management.
```

Comment:
```suggestion
[Semgrep CI](https://github.com/returntocorp/semgrep-action) (aka Semgrep Action or `semgrep-agent`) is a specialized Docker image for running Semgrep in CI environments. It can either be used stand-alone or connected with [Semgrep App](https://semgrep.dev/login) for centralized rule and findings management.
```